app/src/main/java/com/example/smsrc/sms/handlers/SMSSender.java

```diff
@@ -1,4 +1,56 @@
 package com.example.smsrc.sms.handlers;
 
+import android.app.Activity;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.telephony.SmsManager;
+import android.widget.Toast;
+
 public class SMSSender {
+    private static final String SENT = "SMS_SENT";
+    private static final String DELIVERED = "SMS_DELIVERED";
+    private Context context;
+    public SMSSender(Context context){
+        this.context = context;
+    }
+
+    public void send(String text,String phoneNumber){
```

Comment:
Instead of passing text you should pass an SMS.
You can also change SMS to wrap the phone number and only pass an SMS
Note: changing this will affect the listener